2025-05-14 12:57:39,381 ERROR: Erro ao carregar movimentações: 'app.models.ItemMovimentacao object' has no attribute 'apostila_rel' [in C:\Users\nilva\estoque_apostilas\app\routes\movimentacoes.py:222]
Traceback (most recent call last):
  File "C:\Users\nilva\estoque_apostilas\app\routes\movimentacoes.py", line 218, in movimentacao
    return render_template('movimentacoes/movimentacao.html',
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\nilva\AppData\Roaming\Python\Python312\site-packages\flask\templating.py", line 151, in render_template
    return _render(app, template, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\nilva\AppData\Roaming\Python\Python312\site-packages\flask\templating.py", line 132, in _render
    rv = template.render(context)
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\nilva\AppData\Roaming\Python\Python312\site-packages\jinja2\environment.py", line 1295, in render
    self.environment.handle_exception()
  File "C:\Users\nilva\AppData\Roaming\Python\Python312\site-packages\jinja2\environment.py", line 942, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "C:\Users\nilva\estoque_apostilas\app\templates\movimentacoes\movimentacao.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "C:\Users\nilva\estoque_apostilas\app\templates\base.html", line 294, in top-level template code
    {% endfor %} {% endif %} {% endwith %} {% block content %}{% endblock %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\nilva\estoque_apostilas\app\templates\movimentacoes\movimentacao.html", line 151, in block 'content'
    {% for item in mov.itens %} {{ item.apostila_rel.nome }}{% if not
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\nilva\AppData\Roaming\Python\Python312\site-packages\jinja2\environment.py", line 490, in getattr
    return getattr(obj, attribute)
           ^^^^^^^^^^^^^^^^^^^^^^^
jinja2.exceptions.UndefinedError: 'app.models.ItemMovimentacao object' has no attribute 'apostila_rel'
2025-05-14 12:58:10,943 INFO: Acessando rota movimentacao [in C:\Users\nilva\estoque_apostilas\app\routes\movimentacoes.py:194]
2025-05-14 12:58:11,948 ERROR: Erro ao carregar movimentações: 'app.models.ItemMovimentacao object' has no attribute 'apostila_rel' [in C:\Users\nilva\estoque_apostilas\app\routes\movimentacoes.py:222]
Traceback (most recent call last):
  File "C:\Users\nilva\estoque_apostilas\app\routes\movimentacoes.py", line 218, in movimentacao
    return render_template('movimentacoes/movimentacao.html',
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\nilva\AppData\Roaming\Python\Python312\site-packages\flask\templating.py", line 151, in render_template
    return _render(app, template, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\nilva\AppData\Roaming\Python\Python312\site-packages\flask\templating.py", line 132, in _render
    rv = template.render(context)
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\nilva\AppData\Roaming\Python\Python312\site-packages\jinja2\environment.py", line 1295, in render
    self.environment.handle_exception()
  File "C:\Users\nilva\AppData\Roaming\Python\Python312\site-packages\jinja2\environment.py", line 942, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "C:\Users\nilva\estoque_apostilas\app\templates\movimentacoes\movimentacao.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "C:\Users\nilva\estoque_apostilas\app\templates\base.html", line 294, in top-level template code
    {% endfor %} {% endif %} {% endwith %} {% block content %}{% endblock %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\nilva\estoque_apostilas\app\templates\movimentacoes\movimentacao.html", line 151, in block 'content'
    </button>
  File "C:\Users\nilva\AppData\Roaming\Python\Python312\site-packages\jinja2\environment.py", line 490, in getattr
    return getattr(obj, attribute)
           ^^^^^^^^^^^^^^^^^^^^^^^
jinja2.exceptions.UndefinedError: 'app.models.ItemMovimentacao object' has no attribute 'apostila_rel'
2025-05-14 12:58:12,529 INFO: Acessando rota movimentacao [in C:\Users\nilva\estoque_apostilas\app\routes\movimentacoes.py:194]
2025-05-14 12:58:13,348 ERROR: Erro ao carregar movimentações: 'app.models.ItemMovimentacao object' has no attribute 'apostila_rel' [in C:\Users\nilva\estoque_apostilas\app\routes\movimentacoes.py:222]
Traceback (most recent call last):
  File "C:\Users\nilva\estoque_apostilas\app\routes\movimentacoes.py", line 218, in movimentacao
    return render_template('movimentacoes/movimentacao.html',
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\nilva\AppData\Roaming\Python\Python312\site-packages\flask\templating.py", line 151, in render_template
    return _render(app, template, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\nilva\AppData\Roaming\Python\Python312\site-packages\flask\templating.py", line 132, in _render
    rv = template.render(context)
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\nilva\AppData\Roaming\Python\Python312\site-packages\jinja2\environment.py", line 1295, in render
    self.environment.handle_exception()
  File "C:\Users\nilva\AppData\Roaming\Python\Python312\site-packages\jinja2\environment.py", line 942, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "C:\Users\nilva\estoque_apostilas\app\templates\movimentacoes\movimentacao.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "C:\Users\nilva\estoque_apostilas\app\templates\base.html", line 294, in top-level template code
    {% endfor %} {% endif %} {% endwith %} {% block content %}{% endblock %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\nilva\estoque_apostilas\app\templates\movimentacoes\movimentacao.html", line 151, in block 'content'
    </button>
  File "C:\Users\nilva\AppData\Roaming\Python\Python312\site-packages\jinja2\environment.py", line 490, in getattr
    return getattr(obj, attribute)
           ^^^^^^^^^^^^^^^^^^^^^^^
jinja2.exceptions.UndefinedError: 'app.models.ItemMovimentacao object' has no attribute 'apostila_rel'
2025-05-14 12:58:21,753 INFO: Acessando rota movimentacao [in C:\Users\nilva\estoque_apostilas\app\routes\movimentacoes.py:194]
2025-05-14 12:58:22,579 ERROR: Erro ao carregar movimentações: 'app.models.ItemMovimentacao object' has no attribute 'apostila_rel' [in C:\Users\nilva\estoque_apostilas\app\routes\movimentacoes.py:222]
Traceback (most recent call last):
  File "C:\Users\nilva\estoque_apostilas\app\routes\movimentacoes.py", line 218, in movimentacao
    return render_template('movimentacoes/movimentacao.html',
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\nilva\AppData\Roaming\Python\Python312\site-packages\flask\templating.py", line 151, in render_template
    return _render(app, template, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\nilva\AppData\Roaming\Python\Python312\site-packages\flask\templating.py", line 132, in _render
    rv = template.render(context)
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\nilva\AppData\Roaming\Python\Python312\site-packages\jinja2\environment.py", line 1295, in render
    self.environment.handle_exception()
  File "C:\Users\nilva\AppData\Roaming\Python\Python312\site-packages\jinja2\environment.py", line 942, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "C:\Users\nilva\estoque_apostilas\app\templates\movimentacoes\movimentacao.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "C:\Users\nilva\estoque_apostilas\app\templates\base.html", line 294, in top-level template code
    {% endfor %} {% endif %} {% endwith %} {% block content %}{% endblock %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\nilva\estoque_apostilas\app\templates\movimentacoes\movimentacao.html", line 151, in block 'content'
    </button>
  File "C:\Users\nilva\AppData\Roaming\Python\Python312\site-packages\jinja2\environment.py", line 490, in getattr
    return getattr(obj, attribute)
           ^^^^^^^^^^^^^^^^^^^^^^^
jinja2.exceptions.UndefinedError: 'app.models.ItemMovimentacao object' has no attribute 'apostila_rel'
2025-05-14 12:59:14,957 INFO: Estoque de Apostilas startup [in C:\Users\nilva\estoque_apostilas\app\__init__.py:59]
2025-05-14 12:59:36,501 INFO: Acessando rota movimentacao [in C:\Users\nilva\estoque_apostilas\app\routes\movimentacoes.py:194]
2025-05-14 13:06:08,377 INFO: Estoque de Apostilas startup [in C:\Users\nilva\estoque_apostilas\app\__init__.py:59]
2025-05-14 13:06:19,831 INFO: Acessando rota movimentacao [in C:\Users\nilva\estoque_apostilas\app\routes\movimentacoes.py:194]
2025-05-14 15:49:04,417 INFO: Estoque de Apostilas startup [in C:\Users\nilva\estoque_apostilas\app\__init__.py:59]
2025-05-14 15:49:19,323 INFO: Acessando rota movimentacao [in C:\Users\nilva\estoque_apostilas\app\routes\movimentacoes.py:194]
2025-05-14 15:49:52,471 INFO: Acessando rota movimentacao [in C:\Users\nilva\estoque_apostilas\app\routes\movimentacoes.py:194]
2025-05-14 15:49:56,352 INFO: Acessando rota movimentacao [in C:\Users\nilva\estoque_apostilas\app\routes\movimentacoes.py:194]
2025-05-19 02:56:30,160 INFO: Estoque de Apostilas startup [in C:\Users\nilva\estoque_apostilas\app\__init__.py:59]
2025-05-19 03:02:15,469 INFO: Estoque de Apostilas startup [in C:\Users\nilva\estoque_apostilas\app\__init__.py:59]
2025-05-19 03:08:42,105 INFO: Estoque de Apostilas startup [in C:\Users\nilva\estoque_apostilas\app\__init__.py:59]
2025-05-19 03:13:12,360 INFO: Estoque de Apostilas startup [in C:\Users\nilva\estoque_apostilas\app\__init__.py:59]
